.github/workflows/visual_regression_tests.yml

```diff
@@ -0,0 +1,58 @@
+name: "Visual Tests"
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
```

Comment:
Do we need `ready_for_review` ? Or `reopened`? I mean, do we need to trigger the tests if nothing has changed (no diff) ?

Reply:
I'm not sure. Maybe not. I just used this from an example. Why do you think we don't need it?

Reply:
Well, I think we should only trigger vrt when the result of the tests can change. Said differently, is there a scenario where a test will change its output (pass or fail) after marking a PR as ready for review? Or after closing it and opening?
No changes are introduced to any file in those scenarios, so I suppose the test will return the same result? So we will run it twice when we actually don't need to

Reply:
Ok, just saw you added a conditional to only run tests if it's not a draft. Depending on how we work on it I think there might be value on running tests for draft PRs, but I don't feel too strong about this. Let's go with this and see how it plays out